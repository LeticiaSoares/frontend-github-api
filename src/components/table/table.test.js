import React from 'react'
import { shallow } from 'enzyme';
import Table from './index'
import 'jest-styled-components'

describe('<Table />', () => {
    it('should match default snapshot', () => {
        const component = shallow(<Table />)
        expect(component).toMatchSnapshot()
    })
    it('should have the right displayName', () => {
        expect(Table.displayName).toEqual('Table')
    })
})
