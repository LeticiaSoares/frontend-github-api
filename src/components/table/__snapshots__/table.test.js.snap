// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Table /> should match default snapshot 1`] = `ShallowWrapper {}`;
